pykep/python_udpla.cpp

```diff
@@ -39,20 +41,44 @@ python_udpla::python_udpla(py::object obj) : m_obj(std::move(obj))
 };
 
 // Mandatory methods
-[[nodiscard]] std::array<std::array<double, 3>, 2> python_udpla::eph(const kep3::epoch &ep) const
+[[nodiscard]] std::array<std::array<double, 3>, 2> python_udpla::eph(double mjd2000) const
 {
-    auto bf = pykep::callable_attribute(m_obj, "eph");
-    if (bf.is_none()) {
+    auto isthere = pykep::callable_attribute(m_obj, "eph");
```

Comment:
This ``callable_attribute()`` helper directly returns the attribute if it exists, no need to again invoke ``m_obj.attr("eph")`` below. Ditto in several other places where this pattern is used.